cibyl/orchestrator.py

```diff
@@ -292,7 +307,10 @@ def query_and_publish(self, output_style="colorized"):
                 query=get_query_type(**self.parser.ci_args),
                 verbosity=self.parser.app_args.get('verbosity'))
             for system in env.systems:
-                self.run_query(system)
+                if features:
+                    self.run_features(system, features)
+                else:
+                    self.run_query(system)
```

Comment:
I think this is the point where we should start having commands withing cibyl:
- cibyl feature --jobs
- cibyl query --jobs --build

An argument changes the meaning of the command line.

Reply:
It certainly is not. It is some to discuss in the future.